bootstrap.php

```diff
@@ -19,6 +22,64 @@
 	}
 	return;
 }
+
+/*
+ * Initialize coming soon module via container
+ */
+$nfd_module_container = new Container();
+// Set plugin to container
+$nfd_module_container->set(
+	'plugin',
+	$nfd_module_container->service(
+		function() {
+			return new Plugin(
+				array(
+					'id'   => 'web',
+					'file' => WEB_PLUGIN_FILE,
+				)
+			);
+		}
+	)
+);
+// Set coming soon values
+$nfd_module_container->set(
+	'comingsoon',
+	array(
+		'option_name'         => 'mm_coming_soon',
+		'admin_screen_id'     => 'web',
+		'admin_app_url'       => admin_url( 'admin.php?page=web#/home' ),
+		'template_h1'         => 'Coming Soon!',
+		'template_h2'         => 'A New WordPress Site',
```

Comment:
@circlecube We should add translation functions for these two strings.

Reply:
Yes! Thanks